vertx-web/src/main/java/io/vertx/ext/web/impl/Utils.java

```diff
@@ -45,9 +45,14 @@ public static String normalisePath(String path) {
   }
 
   public static String normalisePath(String path, boolean urldecode) {
-    if (path == null || path.charAt(0) != '/') {
-      return null;
+    if (path == null) {
+      return "/";
```

Comment:
Does is introduce a breaking change ? (In other words, should we add a line about this in the breaking change list).